python/dolfinx/fem/__init__.py

```diff
@@ -53,6 +53,6 @@ def create_sparsity_pattern(a: typing.Union[_FormComplex, _FormReal]):
     "assemble_vector_block", "assemble_vector_nest",
     "assemble_matrix_block", "assemble_matrix_nest",
     "assemble_matrix", "set_bc", "set_bc_nest",
-    "DirichletBC", "bcs_by_block", "DofMap", "Form", "IntegralType",
```

Comment:
Using a verb e.g. create_form would make the purpose of the function clearer at the expense of brevity in the code.